docs/docs/reference/other-new-features/extension-methods.md

```diff
@@ -168,38 +168,65 @@ that problem, it is recommended that the name of an extension method is
 preceded by a space and is also followed by a space if there are more parameters
 to come.
 
-### Extension Methods and TypeClasses
+### Extension Methods and Type Classes
 
-The rules for expanding extension methods make sure that they work seamlessly with typeclasses. For instance, consider `SemiGroup` and `Monoid`.
+The rules for expanding extension methods make sure that they work seamlessly with type classes. For instance, consider `SemiGroup` and `Monoid`.
 ```scala
-  // Two typeclasses:
+  // Two type classes:
   trait SemiGroup[T] {
     def (x: T) combine(y: T): T
   }
   trait Monoid[T] extends SemiGroup[T] {
     def unit: T
   }
-
-  // An instance declaration:
-  implicit object StringMonoid extends Monoid[String] {
-    def (x: String) combine (y: String): String = x.concat(y)
-    def unit: String = ""
+  object Monoid {
+    // An instance declaration:
+    implicit object StringMonoid extends Monoid[String] {
+      def (x: String) combine (y: String): String = x.concat(y)
+      def unit: String = ""
+    }
   }
 
   // Abstracting over a typeclass with a context bound:
   def sum[T: Monoid](xs: List[T]): T =
     xs.foldLeft(implicitly[Monoid[T]].unit)(_.combine(_))
 ```
+
 In the last line, the call to `_.combine(_)` expands to `(x1, x2) => x1.combine(x2)`,
 which expands in turn to `(x1, x2) => ev.combine(x1, x2)` where `ev` is the implicit
 evidence parameter summoned by the context bound `[T: Monoid]`. This works since
 extension methods apply everywhere their enclosing object is available as an implicit.
 
+Notice that the `unit` method is not available as an extension method, because it does not have a left-hand side.
+This means we have to write `implicitly[Monoid[T]].unit` to access it, which is rather clunky.
```

Comment:
I would replace this and the next sentence with:

> In order to avoid writing `implicitly[Monoid[T]].unit` to access it, we make `unit` an extension method on the `Monoid` companion object itself as demonstrated below:

Reply:
Made the change.